src/index.ts

```diff
@@ -1,9 +1,12 @@
 import createExplorerLink from './explorer-link'
-import createAccountLink from './account-link'
+import { 
+  getAccountLink as createAccountLink, 
```

Comment:
Nit: We could rename the functions `createAccountLink` and `createCustomAccountLink` respectively, to avoid having to alias them here